web.js

```diff
@@ -1,5 +1,7 @@
 var express = require('express');
 var url = require('url');
+var nodemailer = require('nodemailer');
+var transporter = nodemailer.createTransport();
 
```

Comment:
Generally dynamic IPs can/would be blocked by spam.  We can add an SPF record to reduce the likelihood of the emails getting blocked.  Generally port 25 is blocked from most residential internet provider's ends so local testing is hard on port 25.  We can create an account for aurora and provide authentication, I was on the fence.